### PR TITLE
Content changes on why do you want to be a teacher page

### PR DESCRIPTION
**Pull request by george-bit**
## Context

Change to candidate views as specified inhttps://docs.google.com/document/d/1V-v5ybA7WcOZW5XsI4bA9M47MGov6RGMuqhVEdGqDw8/edit

## Changes proposed in this pull request

Before screenshots are in google doc above
After screen shots:

image
image
image
image

## Guidance to review

review against the requirements in the google doc above, changes are minor but it's just to ratify that I have been faithful to the task specification

## Link to Trello card

https://trello.com/c/LitfW8u4/1842-dev-content-changes-on-why-do-you-want-to-be-a-teacher-page

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/candidate_interface/content/providers.html.erb [hunk 1]
```diff
@@ -6,15 +6,15 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <p class="govuk-body">You can apply to the following training providers and courses using <%= service_name %>. Providers not signed up to <%= service_name %> can only receive applications through UCAS.</p>
-    <h2 class="govuk-heading-s">We suggest you use Apply for teacher training if:</h2>
+    <p class="govuk-body">Not all training providers are signed up to ‘Apply for teacher training’.</p>
```
Comment (paulrobertlloyd) [inline app/views/candidate_interface/content/providers.html.erb:9]:
```suggestion
    <p class="govuk-body">Not all training providers are signed up to ‘<%= service_name %>’.</p>
```

File: app/views/candidate_interface/training_with_a_disability/edit.html.erb [hunk 2]
```diff
@@ -14,16 +14,16 @@
...
           <li>making sure classrooms are wheelchair accessible</li>
         </ul>
-        <p class="govuk-body">If the help you need is not covered by your provider making adjustments, you might also be able to get support from <%= govuk_link_to 'Access to Work', 'https://www.gov.uk/access-to-work' %>. This could include a grant to help cover the costs of practical support in the workplace.</p>
-        <h2 class="govuk-heading-m">It’s against the law to discriminate</h2>
-        <p class="govuk-body">If you’re disabled, <%= govuk_link_to 'it’s against the law to discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>. Training providers must not:</p>
+        <p class="govuk-body">You may also be able to get support from <%= govuk_link_to 'Access to Work', 'https://www.gov.uk/access-to-work' %> to help with workplace adjustments.</p>
+        <p class="govuk-heading-m">It's against the law for training providers to <%= govuk_link_to 'discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```
Comment (paulrobertlloyd) [inline app/views/candidate_interface/training_with_a_disability/edit.html.erb:25]:
```suggestion
        <p class="govuk-heading-m">It’s against the law for training providers to <%= govuk_link_to 'discriminate against you', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```

Comment (george-bit) [reply app/views/candidate_interface/training_with_a_disability/edit.html.erb]: Hi Paul,

I've now committed this suggested change for review.

Thanks

George

File: app/views/candidate_interface/content/providers.html.erb (hunk 1 above)
Comment (george-bit) [reply app/views/candidate_interface/content/providers.html.erb]: Hi Paul,

I've now committed this suggested change for review.

Thanks

George

**Comment by EmmaFrith** [comment]
Hi @tvararu @george-bit - I didn't ask for the changes which have be made and deployed. Can you revert these changes please, to all 4 pages? I think there has been a misunderstanding with the work that was supposed to be done. I provided content for this work, but the content hasn't been used. We also haven't iterated the content for the disability page, so I'm not sure why that has been picked up.

**Pull request merged**